Jenkinsfile

```diff
@@ -30,7 +30,7 @@ try {
                     // The positions of the slashes are important here:
                     // There should be a slash after _site BUT not after
                     // public_html.
-                    sh 'rsync -avzpce "ssh -o StrictHostKeyChecking=no" --del _site/ --exclude static decal@ssh.ocf.berkeley.edu:public_html'
+                    sh 'rsync -avzpce "ssh -o StrictHostKeyChecking=no" --del _site/ --exclude static ocfdecal@decal.ocf.berkeley.edu:/srv/www/decal/'
```

Comment:
Is index.html supposed to end up in the `decal` folder? If yes, there shouldn't be a slash at the end of this line.